Menus/SettingsInputMappingGUI.cpp

```diff
@@ -0,0 +1,214 @@
+#include "SettingsInputMappingGUI.h"
+#include "UInputMan.h"
+
+#include "GUI.h"
+#include "AllegroBitmap.h"
+#include "GUICollectionBox.h"
+#include "GUIButton.h"
+#include "GUIScrollbar.h"
+#include "GUILabel.h"
+
+namespace RTE {
+
+	std::array<InputElements, 7> SettingsInputMappingGUI::m_InputElementsUsedByMouse = { InputElements::INPUT_FIRE, InputElements::INPUT_PIEMENU, InputElements::INPUT_AIM, InputElements::INPUT_AIM_UP, InputElements::INPUT_AIM_DOWN, InputElements::INPUT_AIM_LEFT, InputElements::INPUT_AIM_RIGHT };
```

Comment:
You can do all of that by directly fiddling with the map in ini, as you always could.
This is here to protecc you from being dumb and breaking your mouse mappings because there aren't and never were any mouse mapping controls.
Will be gone once mouse mapping is implemented.

Reply:
Okay, that's fine then. Having mouse mapping implemented and removing this in future would be good but it sure doesn't have to be done in this monster pr. Feel free to resolve and/or makeissue:tm: